src/controllers/action-queue/runtime.ts

```diff
@@ -33,36 +33,36 @@ export const executeActionProgram = async (context: ExecutionContext, program: A
     try {
       const dryrunOutputs = await dryrunActionProgram(context, program, state, true)
 
-      // Convert each dryrun result into an array of push events for the push-server.
-      const newPushEvents = await prepareNewPushEvents(context, program, effect, dryrunOutputs)
-
-      // Send PushEvents to the push server:
-      await uploadPushEvents(context, { createEvents: newPushEvents })
-
-      // Mutate the nextState accordingly; effect should be awaiting push events:
-      const nextChildEffects = newPushEvents.map(event => ({
-        type: 'push-event',
-        eventId: event.eventId
-      }))
-      let nextEffect: ActionEffect
-      if (effect.type === 'seq') {
-        // Drop the last effect because it is to be replaced by the first push-event effect
-        const prevOpIndex = effect.opIndex // Same opIndex because the first of nextChildEffects replaces the last or prevChildEffects
-        const prevChildEffects = effect.childEffects.slice(0, -1) // Slice to drop the last of prevChildEffects
-        nextEffect = {
-          type: 'seq',
-          opIndex: prevOpIndex,
-          // @ts-expect-error
-          childEffects: [...prevChildEffects, ...nextChildEffects]
+      if (dryrunOutputs.length > 0) {
```

Comment:
Is there some way to check if the dryrunOutputs are invalid and throw?

Reply:
Invalid in what way?

Reply:
Is there some way to check if something went wrong with dryrunOutput generation?